packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs

```diff
@@ -68,31 +67,27 @@ pub fn rest_api_router<P: GraphPool>(store: Arc<P>) -> Router {
             get({
                 let doc = open_api_doc;
                 move || async { Json(doc) }
-            })).route("/models/*path", get(static_schemas)),
+            })).route("/models/*path", get(serve_static_schemas)),
         )
 }
 
 #[allow(
     clippy::unused_async,
     reason = "This route does not need async capabilities, but axum requires it in trait bounds."
 )]
-async fn static_schemas<'a>(Path(path): Path<String>) -> impl IntoResponse {
+async fn serve_static_schemas(Path(path): Path<String>) -> Result<Response, StatusCode> {
```

Comment:
I'm not quite sure about the naming of this function, what is it actually doing? It's building a response with a given schema or something?

The actual function doesn't seem to "serve" anything, and also from how I understand it, it does not serve schema _s_, but just a specific one.

Reply:
It serves the JSON Schemas we define explicitly in the `./json_schemas` folder served at `http://localhost:3000/api-doc/models/*`. 

Why doesn't it feel like it's serving anything? It serves everything in that folder for the user as a static path

Reply:
I guess it just seems like the function is more of a builder or constructor or something. The web framework is doing the serving, this function is not. That said, I'm not really coming up with a better name 🙃 

I think we still need to figure something else out though as the name is a bit misleading, per call it is about _a_ schema, not all static schemas.

Reply:
There's another project using the same setup calling this [`serve_asset` ](https://github.com/euclio/aurelius/blob/bcfafde6059efa7f55e787dd6e89759be0fb3f94/src/service.rs#L27) It seems to be of convention to call these routes something along those lines. I agree that it should be singular, though

Reply:
If we change it to singular I'm happy to approve then

Reply:
Addressed in d41bfded81b21b6d2399dc4ae3bdc9139f3b55a9